components/topbar/user/src/index.js

```diff
@@ -347,6 +348,11 @@ TopbarUser.propTypes = {
       })
     )
   }).isRequired,
+  /**
+   * Render custom content instead of a CTA.
+   * If a customContent is provided, it's rendered instead of a navCTA
+   */
+  customContent: PropTypes.node,
```

Comment:
🍆 just wondering... would you see any benefit in using `children` instead of this `customContent` prop?

Reply:
IMHO children should be used when you want to place something inside, it's a too strict API. @jordevo

Reply:
fair enough! 👍